src/components/overlayCookie/overlayCookie.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+import 'i18n/i18n';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import styles from './overlayCookie.scss';
+
+class OverlayCookie extends Component {
+  state = {
+    cookiesAccepted: 'false'
+  };
+
+  componentDidMount(){
+    this.checkCookiePermission();
+  }
+
+  render() {
```

Comment:
Just for consistency with the rest of our codebase, can you move the `render` method to the end of the class? Just after the `checkCookiePermission` method.